src/apps/lwaftr/binding_table.lua

```diff
@@ -65,7 +65,7 @@ module(..., package.seeall)
 
 local bit = require('bit')
 local ffi = require("ffi")
-local stream = require("apps.lwaftr.stream")
+local lwstream = require("apps.lwaftr.stream")
```

Comment:
In general, I agree. In this module, however, the `stream` identifier is used 3 times as a library reference, and 17 times as a local variable. I'd like to disambiguate the two: what do you suggest?